substrate/.maintain/frame-weight-template.hbs

```diff
@@ -16,6 +16,7 @@
 #![allow(unused_parens)]
 #![allow(unused_imports)]
 #![allow(missing_docs)]
+#[allow(dead_code)]
```

Comment:
let's write the same in the other file: `substrate/.maintain/frame-umbrella-weight-template.hbs`

Reply:
@gui1117 done [5fa92d3](https://github.com/paritytech/polkadot-sdk/pull/7408/commits/5fa92d3a5728e0a82599749b4620e8154c829beb)